app/macos/main.js

```diff
@@ -16,9 +17,15 @@ const path = require("path");
 
 const assetsDirectory = path.join(__dirname, "assets");
 
+// JSON list of emojis
+const emojis = require("./src/emojis");
+
 let tray = undefined;
 let window = undefined;
 
+// Let's preset our LRU Map that we'll use to keep track of recent uses
+let lruMap = new LRUMap(10);
```

Comment:
We want the LRU Map in main.js instead of index.js to persist the object across its entire runtime, not restart every time the modal pops up.